core/geometry/src/test/polyface/Polyface.test.ts

```diff
@@ -127,394 +125,299 @@ function exercisePolyface(ck: Checker, polyface: Polyface, longEdgeIsHidden: boo
-      const delta = maxRange.maxAbs();
-      const delta10 = 10 * delta;
-      let x = 0;
-      for (let i = 0; i < testCase.data.length; ++i) {
-        for (let j = i + 1; j < testCase.data.length; ++j) {
-          let y = 0;
-          // lambda to exercise local range clash methods
-          const clashDetect = (index0: number, index1: number, captureLocal: boolean = false, captureWorld: boolean = true, captureIntersection: boolean = true): boolean => {
-            if (captureLocal) {
-              // doLocalRangesIntersect converts range0 to a polyface transformed into range1's local coordinates
-              GeometryCoreTestIO.captureTransformedRangeEdges(allGeometry, testCase.data[index0].localRange, testCase.data[index1].localToWorld.inverse()?.multiplyTransformTransform(testCase.data[index0].localToWorld), x, y, z);
-              GeometryCoreTestIO.captureRangeEdges(allGeometry, testCase.data[index1].localRange, x, y, z);
-            }
-            if (captureWorld) {
-              GeometryCoreTestIO.captureTransformedRangeEdges(allGeometry, testCase.data[index0].localRange, testCase.data[index0].localToWorld, x, y, z);
-              GeometryCoreTestIO.captureTransformedRangeEdges(allGeometry, testCase.data[index1].localRange, testCase.data[index1].localToWorld, x, y, z);
-            }
-            const isClash = ClipUtilities.doLocalRangesIntersect(testCase.data[index0].localRange, testCase.data[index0].localToWorld, testCase.data[index1].localRange, testCase.data[index1].localToWorld);
-            ck.testBoolean(isClash, testCase.expectedClash, `ranges clash as expected: i=${index0} j=${index1}`);
-            const clashRange = ClipUtilities.rangeOfIntersectionOfLocalRanges(testCase.data[index0].localRange, testCase.data[index0].localToWorld, testCase.data[index1].localRange, testCase.data[index1].localToWorld);
-            if (captureIntersection)
-              GeometryCoreTestIO.captureRangeEdges(allGeometry, clashRange, x, y, z);
-            if (ck.testTrue(isClash === !clashRange.isNull, "intersection range is non-null iff ranges clash") && isClash)
-              ck.testRange3d(clashRange, testCase.clashRange![Math.min(index0, index1)][Math.max(index0, index1)]!, "intersection has expected world range");
-            return isClash;
-          };
-
-          const clashIJ = clashDetect(i, j);
-          y += delta10;
-          const clashJI = clashDetect(j, i);
-          ck.testBoolean(clashIJ, clashJI, `symmetric arguments: i=${i} j=${j}`);
-
-          // cover the margin case, no output or test
-          ClipUtilities.doLocalRangesIntersect(testCase.data[i].localRange, testCase.data[i].localToWorld, testCase.data[j].localRange, testCase.data[j].localToWorld, 1.0);
-          x += delta10;
-        }
-      }
-      z += delta10;
-    }
-    GeometryCoreTestIO.saveGeometry(allGeometry, "Polyface", "IntersectLocalRangeBoxes");
-    expect(ck.getNumErrors()).equals(0);
-  });
```

Comment:
"Polyface.dihedralAngleSummary" is a new test I added for behavior change of "dihedralAngleSummary"

Reply:
This test should go in PolyfaceQuery.test.ts.  Please also check that `PolyfaceQuery.dihedralAngleSummary(polyface, true) === 0` as this is a planar mesh.

Reply:
Done